SVE2 support breaks Android build

```
[arm64-v8a] Compile : SDL3 <= SDL_sve2_blit_N.c
In file included from ./src/video/arm/SDL_sve2_blit_N.c:50:
In file included from ./src/video/arm/SDL_sve2_swizzle.h:31:
./src/video/arm/SDL_sve2_extension.h:730:15: warning: unsupported architecture 'armv8-a+sve2' in the 'target' attribute string; 'target' attribute ignored [-Wignored-attributes]
SDL_TARGETING("arch=armv8-a+sve2")
```

--- a/docs/README-android.md
+++ b/docs/README-android.md
@@ -13,7 +13,7 @@ Requirements
 Android SDK (version 35 or later)
 https://developer.android.com/sdk/index.html
 
-Android NDK r15c or later
+Android NDK r28c or later
 https://developer.android.com/tools/sdk/ndk/index.html
 
 Minimum API level supported by SDL: 21 (Android 5.0)

